### PR TITLE
As an SDK user, I want to be able get app information from the SDKs

### PR DESCRIPTION
**Pull request by calhorn95**
add GetApp functionality
 - add test to app_test.go

### CODE REVIEW

**Review by dylanbrookes — APPROVED** [review]
Looks good - thoughts on adding descriptions to each of the struct fields for intellisense support?

image

**Comment by calhorn95** [comment]
yes I can add that

File: app.go [hunk 1]
```diff
@@ -72,6 +72,90 @@ type MagicLink struct {
...
+// returns App on success, error on failure
+func (a *App) GetApp() (*AppInfo, error) {
+	type respAppInfo struct {
+		App AppInfo `json:"app"`
+	}
+	var appResp respAppInfo
+
+	response, err := resty.New().R().
+		SetResult(&appResp).
+		SetAuthToken(a.Config.APIKey).
```
Comment (dylanbrookes) [inline app.go]: Is there a reason we're setting an auth token for an unprotected route? We don't need an authorization header to access the endpoint:
`https://api.passage.id/v1/apps/{appID}`

**Review by dylanbrookes — CHANGES_REQUESTED** [review]

File: app.go (hunk 1 above)
Comment (calhorn95) [reply app.go]: good call

**Review by dylanbrookes — APPROVED** [review]
Looks good now - good stuff on the struct descriptions

**Pull request merged**